API.md

```diff
@@ -24,6 +24,8 @@ The following creates a basic static file server that can be used to serve html
 const Path = require('path');
 const Hapi = require('@hapi/hapi');
 const Inert = require('@hapi/inert');
+// In ESM, may import as:
+// import { inertPlugin } from '@hapi/inert';
```

Comment:
Why? `import Inert from '@hapi/inert'` works fine when passed to `register()`. And if you want to use the `plugin` you can always do: `import { plugin as InertPlugin } from '@hapi/inert';`

Reply:
Good point, I overthought this 👍